### ISSUE TITLE
Labels are all wrong

### ISSUE THREAD

**Issue by woundenfish** [khaclinh/pp4av#1]
...

**Comment by khaclinh** [comment]
Dear @woundenfish,
Could you share any example that you got wrong label?

**Comment by AndreiMoraru123** [comment]
Well, some are. E.g:

![wrong](https://github.com/khaclinh/pp4av/assets/81184255/338bb515-f90c-4c5a-a44b-28b973dd84c9)

Definitely not all, though, as the issue suggests. I would say most of them are okay by randomly sampling a bit:

![correct](https://github.com/khaclinh/pp4av/assets/81184255/9ccfd472-f85e-48b7-b993-3756b7348f6b)

These were downloaded and meshed together from the [hugginface repo](https://huggingface.co/datasets/khaclinh/pp4av/tree/main/data).

**Comment by khaclinh** [comment]
Sorry for my late response.

I have double checked these above samples.
The first sample is the image images/paris/frame_001700.png in pp4av dataset.
The annotation did not consists of any face as in your plot. Here is the annotation in annotation file (annotations/paris/frame_001700.png):
1 0.725547 0.736667 0.031094 0.015556
1 0.563672 0.704306 0.007500 0.009722
1 0.970000 0.927917 0.060313 0.036389
1 0.509609 0.702569 0.025469 0.011806
Note that class 1 is denoted for License Plate. Here is it's visualization:
![image](https://github.com/khaclinh/pp4av/assets/6778522/93d3f679-fd2b-4f4a-b9fa-e997f921194c)

I think you did wrong plot. Highly suggest you to use CVAT tool (https://github.com/opencv/cvat) to load data and annotation for visualization.
The thing happened the same in the second image.

I hope this answer will help for your work.
If you still face any further issue regarding to the dataset, please feel free to let me know.
Thanks,

**Comment by AndreiMoraru123** [comment]
Hey @khaclinh, thanks for clarifying!

Yea, definitely looks like I got that one plot wrong, but it's a bit weird some of them were okay. I was trying to convert them to [`CocoDetection`](https://pytorch.org/vision/stable/generated/torchvision.datasets.CocoDetection.html) format before training a detector on them. They did turn out okay I believe:

https://github.com/AndreiMoraru123/DetrPP4AV/tree/main

**Comment by khaclinh** [comment]
Sounds good. Thanks @AndreiMoraru123  for your sharing.
